internal/constants/constants.go

```diff
@@ -49,3 +49,24 @@ const (
 
 	Qwatch string = "qwatch"
 )
+
+var IgnoreCommands = map[string]string{
+	"SELECT":    "ignore for tcl test",
+	"FUNCTION":  "ignore for tcl test",
+	"FLUSHALL":  "ignore for tcl test",
+	"RPUSH":     "ignore for tcl test",
+	"HSET":      "ignore for tcl test",
```

Comment:
We do support HSET in dice

Reply:
Thanks @JyotinderSingh for mentioning. I mistakenly added HSET instead of HGET. Updated.